projects/observability/src/pages/custom-dashboards/custom-dashboard-panel-edit/custom-dashboard-panel-edit.component.ts

```diff
@@ -19,9 +19,12 @@ import { SPAN_SCOPE } from '../../../shared/graphql/model/schema/span';
 import { MetadataService } from '../../../shared/services/metadata/metadata.service';
 import { ExplorerGeneratedDashboard } from '../../explorer/explorer-dashboard-builder';
 import { getLayoutForElements } from '../../explorer/utils/get-layout-for-elements';
-import { CustomDashboardStoreService, PanelData } from '../custom-dashboard-store.service';
+import { CustomDashboardStoreService } from '../custom-dashboard-store.service';
```

Comment:
Why do we have two different import lines for `CustomDashboardStoreService` and `PanelData`?

Reply:
Generally a good practice to separate type imports. Haven't done an analysis on how Angular handles these but there can only be benefits from separating type imports, no harm. Benefits for various tools are listed [here][1].

Needed to disable the lint rule on the line because TSLint is outdated and doesn't differentiate between regular and type imports. The new [replacement rule][2] does this differentiation out of the box.

[1]: https://stackoverflow.com/a/64243357/7435656
[2]: https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/no-duplicates.md#importno-duplicates

Reply:
Update - Just found this:
https://github.com/angular/angular/pull/37912
Angular differentiates them too. TSLint is lagging behind.

Reply:
Missed that you had imported `type` there. Makes sense.